management-packages/core-management-cliplugins/bundle/config/01_cliplugin-crd.yaml

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
**question**: This feels a little odd, but I can see benefits to the pattern, and I'm not up to speed on what we're doing elsewhere. Is duplicating this CRD YAML a best practice? Or should we somehow reference the existing file in the repo? (Or copy that file here as a part of the build, or something?)

Reply:
So, we are doing the same for the `featuregates` package for example https://github.com/vmware-tanzu/tanzu-framework/blob/main/management-packages/featuregates/bundle/config/upstream/feature-crd.yaml.

But yes, something like this can be referenced as well. But we might need to add some tooling to make sure we build the CRDs and copy the files. I can file a separate issue to track this effort as it applies to other package as well.